source/install-guide/index.rst

```diff
@@ -1,21 +1,22 @@
-.. _installing:
+.. _installation:
 
-Installing the application
-##########################
+Installation guide
+##################
 
 .. versionadded:: 3.2
 
-**Important: GeoNetwork requires a Java 8 environment (JRE) to be installed on your system. Later versions of java will not work at present. This must be done prior to installation.**
+**Important: GeoNetwork requires a Java 8 environment (JRE) to be installed on your system. Later versions of Java will not work at present. This must be done prior to installation.**
 
 
-Before using |project_name| on your own computer, you need to install it.
-There is different ways to install |project_name|:
+Before you can use GeoNetwork on your own computer, you need to install it.
+There are several different ways to install GeoNetwork:
 
 
 .. toctree::
    :maxdepth: 2
 
    installing-from-the-installer.rst
```

Comment:
I would remove that. The website download page https://geonetwork-opensource.org/downloads.html can have link to some other packages

Reply:
As mentioned above, I will remove it and point to the download section on the GeoNetwork website.
Shortly, I will therefore make a PR for the GeoNetwork website repo as well.

Reply:
Done: https://github.com/geonetwork/website/pull/28